### PR TITLE
docs: add namespace information for instascale-ocm-secret

### PR DESCRIPTION
**Pull request by VanillaSpoon**
# What changes have been made
This pr contains a brief update to the readme to ensure the instascale-ocm-secret is set in the correct namespace.

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [x] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,13 +38,14 @@ Key features:
 - Deploy MCAD using steps [here](https://github.com/project-codeflare/multi-cluster-app-dispatcher/blob/main/doc/deploy/deployment.md).
 
 - In Visual Studio Code update `.vscode/launch.json` so that `"KUBECONFIG"` points to your Kubernetes config file.<br>
-- If you changed the namespace in `instascale-configmap.yaml` update the `args[]` in `launch.json` to include `--configs-namespace="<YOUR_NAMESPACE>"`.<br>
+- If you changed the namespace in `instascale_configmap.yaml` update the `args[]` in `launch.json` to include `--configs-namespace="<YOUR_NAMESPACE>"`.<br>
```
Comment (astefanutti) [inline README.md:41]: I think the proper way of changing the namespace, when InstaScale is deployed with `make deploy`, is to have it changed globally here:

https://github.com/project-codeflare/instascale/blob/cc2a63c5535de89ed38d43746251bc3c9bc3c9ce/config/default/kustomization.yaml#L2

Comment (VanillaSpoon) [reply README.md:41]: Sorry that's right, great spot Antonin :)

Comment (VanillaSpoon) [reply README.md]: Thanks @astefanutti, 
This has been updated

Comment (astefanutti) [reply README.md]: Agreed, this can be re-worked later if needed.

**Pull request merged**